pages/dkp/kommander/2.3/clusters/custom-domain-certificate/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle: Custom domain and certificate
+title: Configure a custom domain and certificate
+menuWeight: 30
+excerpt: Configure a custom domain and certificate for your Management or any Managed/Attached cluster
+beta: false
+enterprise: false
+---
+
+DKP supports configuring a custom domain name for accessing the DKP UI and other platform services. Additionally, you can provide a custom certificate for each domain, or one can be issued automatically by Let's Encrypt, or other certificate authorities supporting the ACME protocol.
+
+The configuration path is different depending on the cluster for which you want to customize the domain or certificate.
+
+[Customize a domain or certificate in the **Management Cluster**][management] (during installation).
+
+[Customize a domain or certificate in a **Managed Cluster**][managed] (after installation).
+
+## Why to customize domains
+
+DKP’s default configuration supports the customization of domains to allow you to use your own domain or hostname for your services.
+
+For example, you can set up your DKP UI to be accessible via the provided domain instead of the URL provided by default. You can also use your domain to enable your company to access back end services, or your customers to access front end services via a specific domain. Or you can configure a custom domain to make a cluster accessible with a specific domain.
```

Comment:
> DKP UI to be accessible via the provided **domain** instead of the **URL** provided by default.

We should use a single term for domain across the whole document.

Reply:
```suggestion
For example, you can set up your DKP UI to be accessible via your custom domain name instead of the domain provided by default. You can also use your domain to enable your company to access back-end services, or your customers to access front-end services via a specific domain. Or you can configure a custom domain to make a cluster accessible via a specific domain.
```